securedrop/static/js/source.js

```diff
@@ -122,6 +122,14 @@ function suggestTor() {
   addFadingClose("use-tor-browser-close", useTorBrowser);
 }
 
+function checkClearnet() {
+  let url = new URL(location.href);
+  // Allow localhost for development
+  if (url.hostname != "127.0.0.1" && !url.host.endsWith(".onion")) {
```

Comment:
minor point: can `localhost` be added to the allowed hostnames?

Reply:
Done.